src/backend/crossterm.rs

```diff
@@ -1,20 +1,19 @@
-//! This module provides the `CrosstermBackend` implementation for the `Backend` trait.
-//! It uses the `crossterm` crate to interact with the terminal.
+//! This module provides the [`CrosstermBackend`] implementation for the [`Backend`] trait. It uses
+//! the [Crossterm] crate to interact with the terminal.
```

Comment:
Is there a reason `Crossterm` doesn't have backticks?

```suggestion
//! the [`Crossterm`] crate to interact with the terminal.
```

Reply:
Never mind, I see the link at bottom now.